tests/stdlib/thttpclient.nim

```diff
@@ -41,6 +41,10 @@ proc asyncTest() {.async.} =
   var client = newAsyncHttpClient()
   var resp = await client.request("http://example.com/", HttpGet)
   doAssert(resp.code.is2xx)
+  doAssert(resp.request.reqMethod == HttpGet)
+  doAssert($(resp.request.url) == "http://example.com/")
```

Comment:
```suggestion
  doAssert $resp.request.url == "http://example.com/"
```

(. has precedence over $)

(and uses the more idiomatic doAssert x instead of doAssert(x))

ditto elsewhere

Reply:
@timotheecour helpful tips, thanks